tests/integration/call_run_within_script_with_keras_fit.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Example from TF tutorials
+# https://www.tensorflow.org/tutorials/distribute/keras
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import tensorflow_datasets as tfds
+import tensorflow as tf
+
+from tensorflow_cloud import machine_config
+from tensorflow_cloud import run
+
+tfds.disable_progress_bar()
+
+print(tf.__version__)
+
+# Calling `run` from within a script with contains the Keras model.
+run.run(
```

Comment:
UX note: it's weird to have `run.run()`. I would expect:

```
import tensorflow_cloud as tfc
tfc.run(...)
```

Reply:
Good point. Updated the imports.